static/app/views/newsletterConsent.tsx

```diff
@@ -22,35 +23,37 @@ export default class NewsletterConsent extends Component<Props> {
   render() {
     return (
       <NarrowLayout>
-        <p>
-          {t('Pardon the interruption, we just need to get a quick answer from you.')}
-        </p>
-
         <ApiForm
           apiMethod="POST"
           apiEndpoint="/users/me/subscriptions/"
           onSubmitSuccess={() => this.props.onSubmitSuccess?.()}
           submitLabel={t('Continue')}
         >
-          <RadioBooleanField
-            key="subscribed"
+          <p>
+            {t('Pardon the interruption, we just need to get a quick answer from you.')}
```

Comment:
nit: should we replace `Pardon` with `Sorry`? :D

Reply:
We could, but the phrasing would need to change to something like `Sorry to interrupt, we just...`.

Reply:
wouldn't it be better? :)